securedrop/crypto_util.py

```diff
@@ -259,6 +260,8 @@ def decrypt(self, secret, ciphertext):
         """
         hashed_codename = self.hash_codename(secret,
                                              salt=self.scrypt_gpg_pepper)
+        if six.PY2:  # Python2
```

Comment:
This is a stylistic comment only. This duplicates the logical line of code. I think it would make more sense to do:

```python
data = self.gpg.decrypt(ciphertext, passphrase=hashed_codename).data

if not six.PY2: # in python 3 the output is bytes, but we need str
    data = data.decode('utf-8')

return data
```

This makes it easier to spot the python2/3 differences at a quick glance.

Reply:
Doing this.